server/server.js

```diff
@@ -59,9 +59,11 @@ app.get('/auth/github/callback',
   passport.authenticate('github', { failureRedirect: '/login' }),
   function(req, res) {
   	req.session.username = req.user.username;
+    req.session.avatar_url = req.user.avatar_url;
 
     // Store github cookie for 7 days
     res.cookie('github', req.user.username, { expires: new Date(Date.now() + (1000 * 60 * 60 * 24 * 7))});
+    res.cookie('github.avatar', req.user._json.avatar_url, { expires: new Date(Date.now() + (1000 * 60 * 60 * 24 * 7))});
 
```

Comment:
Could we create only one cookie with all the information needed (+ user id) instead of creating several cookies?